src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java

```diff
@@ -31,10 +32,14 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.net.URISyntaxException;
+
 /**
  * Represents Cdf Studio Page Actions
  */
 public class CdfStudioActions {
+
+  public static String pipelineName;
```

Comment:
nit: remove empty lines and unused variable

Reply:
Removed.